install.sh

```diff
@@ -104,18 +152,15 @@ if [ ! -f device.json ]; then
     --arg key1 'installed_packages' \
     '. | .[$key0]=$value0 | .[$key1]=[]' <<<'{}' > device.json
 fi
-echo -e "\n${YELLOW}Downloading information for Bootstrap packages...${RESET}"
-echo -e "${GRAY}"
-(cd "${CREW_LIB_PATH}"/packages && curl -#OL "${URL}"/packages/{"${BOOTSTRAP_PACKAGES// /,}"}.rb)
-echo -e "${RESET}"
 
 for package in $BOOTSTRAP_PACKAGES; do
-  pkgfile="$(cat "${CREW_LIB_PATH}"/packages/"$package".rb)"
-  temp_url="$(echo "$pkgfile" | grep -m 3 "$ARCH": | head -n 1 | cut -d\' -f2 | tr -d \' | tr -d \" | sed 's/,//g')"
-  temp_sha256="$(echo "$pkgfile" | grep -m 3 "$ARCH": | tail -n 1 | cut -d\' -f2 | tr -d \' | tr -d \" | sed 's/,//g')"
-  urls[k]="$temp_url"
-  sha256s[k]="$temp_sha256"
-  k=$((k+1))
+  pkgfile="${CREW_PACKAGES_PATH}/${package}.rb"
+
+  [[ "$(sed -n '/binary_sha256/,/}/p' "${pkgfile}")" =~ .*x86_64:[[:blank:]]*[\'\"]([^\'\"]*) ]]
+    sha256s+=("${BASH_REMATCH[1]}")
+
+  [[ "$(sed -n '/binary_url/,/}/p' "${pkgfile}")" =~ .*x86_64:[[:blank:]]*[\'\"]([^\'\"]*) ]]
```

Comment:
Shouldn't we abstract the arch with `$ARCH` instead of `x86_64`?

Reply:
Change made